Test/Test.csproj

```diff
@@ -1,14 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
```

Comment:
this is one of the changes that I'm not entirely following. Did you make this change or was it made by visual studio?

Reply:
Instead of adding `-f netcoreapp2.1` to every `dotnet test` in every CI build
I configured the project to target .NET Core always
```
<TargetFrameworks>netcoreapp2.2</TargetFrameworks>
```
and additionaaly target .NET Framework when running under Windows
```
<TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
```